test/system_settings_test.rb

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "SystemSettings"
+
+describe "Yast::SystemSettings" do
+  Yast.import "Bootloader"
```

Comment:
NP: Any reason to have the `import` here? Could you move it above so both `import` calls are together?